src/WorkQueue/ColumnDefs/NPH/config.json

```diff
@@ -0,0 +1,172 @@
+      "sortable": false,
+      "filterable": false,
+      "dataField": "firstName",
+      "workqueueField": "firstname",
+      "class": "participantLink",
+      "group": "Participant Details"
+    },
+    {
+      "display": "Last Name",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "lastName",
+      "workqueueField": "lastname",
+      "class": "participantLink",
+      "group": "Participant Details"
+    },
+    {
+      "display": "Middle Name",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "middleName",
+      "workqueueField": "middleinitial",
+      "class": "participantLink",
+      "group": "Participant Details"
+    },
+    {
+      "display": "Participant ID",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "participantNphId",
+      "workqueueField": "participantid",
+      "class": "participantLink",
+      "group": "Participant Details"
+    },
+    {
+      "display": "Enrollment Status",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphEnrollmentStatus",
+      "class": "enrollmentStatus",
+      "group": "Activity Status"
+    },
+    {
+      "display": "Withdrawal Status",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphWithdrawalStatus",
+      "class": "statusField",
+      "group": "Activity Status",
+      "enable": false
+    },
+    {
+      "display": "Deactivation Status",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphDeactivationStatus",
+      "class": "statusField",
+      "group": "Activity Status",
+      "enable": false
+    },
+    {
+      "display": "Module 1 Consent Status",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphModule1ConsentStatus",
+      "class": "moduleConsentStatus",
+      "group": "Consents"
+    },
+    {
+      "display": "Module 1 Hair and Nail Opt In",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphModule1ConsentStatus",
+      "class": "module1HairAndNailOptIn",
+      "group": "Consents"
+    },
+    {
+      "display": "Module 1 Recontact Opt In",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphModule1ConsentStatus",
+      "class": "module1RecontactConsent",
+      "group": "Consents"
+    },
+    {
+      "display": "Module 2 Consent Status",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphModule2ConsentStatus",
+      "class": "moduleConsentStatus",
+      "group": "Consents",
+      "enable": true
+    },
+    {
+      "display": "Module 3 Consent Status",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphModule3ConsentStatus",
+      "class": "moduleConsentStatus",
+      "group": "Consents",
+      "enable": true
+    },
+    {
+      "display": "Paired Site",
+      "sortable": false,
+      "filterable": false,
+      "dataField": "nphPairedSite",
```

Comment:
We might need to display the site name instead of the site ID. You could use the getNphSiteDisplayName method in the SiteServe class to implement this.